.circleci/config.yml

```diff
@@ -25,8 +26,8 @@ jobs:
       - run: if [ "$LIVEBRANCHES" = true ]; then ./wait-for-running-branch.sh; fi
       - run: cd wp-e2e-tests && npm run decryptconfig
       - run: sudo chmod +x wp-e2e-tests/node_modules/.bin/magellan
-      - run: echo running test command "./run.sh -R ${testFlag--C} $RUN_ARGS"
-      - run: cd wp-e2e-tests && ./run.sh -R ${testFlag--C} $RUN_ARGS
+      - run: echo running test command "./run.sh -p -R ${testFlag} $RUN_ARGS"
```

Comment:
do you want to default this to `-g` so when this isn't triggered by the API (eg. a change to the config) it has something to run? Or does it does this and I'm missing something...

Reply:
The `-p`, according to the run.sh notes implies `-g -s mobile,desktop` so we're already covered.

Reply:
Oh thanks 👍